### DISCUSSION TITLE
Github Mobile isn't open source

### DISCUSSION THREAD

**Discussion by gizmovt** [community/community#2841] [Mobile]
I did a search on this feedback section to see if this was even a question. I'm not sure why this hasn't been asked before (or at least that I couldn't find it) but: Why isn't the mobile app for the effective face of the open source community **not** open source?

It just seems ironic to me that an application designed by the company that is leading the open source community hasn't decided to do so themselves. Is this an IP issue? I really haven't been able to find much discussing such other than knowing that the current answer is "Maybe".

**Comment by tentacleuno** [comment]
Oh dear. You'd think they would open-source such an app, but clearly not. Is there any official response from GitHub regarding this matter?

**Comment by gladiatr72** [comment]
Microsoft.

Really.

You opened a github issue to try to railroad Microsoft into open sourcing github mobile? 

M$ w/out Gates/Balmer is still M$. 

Good luck. Don't pass up any interesting opportunities in the mean time.

**Comment by tvvocold** [comment]
+1, ping @nat

**Comment by flleeppyy** [comment]
Any open source github clients?

**Reply by hamishmb** [reply]
Octodroid for Android is on F-Droid :)

**Reply by XMA2l** [reply]
> Octodroid for Android is on F-Droid :)

Needed to install this proprietary thing to find out - thx a lot :)

**Comment by hamishmb** [comment]
+1 And by the way Octodroid (on F-Droid) is a good FOSS client for Android

**Comment by R3AP3R28011990** [comment]
I'll give an answer you should get octodroid on F Droid 
It is an excellent FOSS Client for Android

**Comment by Wladefant** [comment]
That should be really considered, because you can get all the benefits of a company baked open source projects and that is normally a win win situation for everyone. Bug reports could be filed faster and directly in association tk the code...

**Comment by T-vK** [comment]
FastHub-Libre is another good open source client found on F-Droid.

Very disappointed to see GitHub hiding their source code. One can only wonder what they're trying to hide. 
Good thing Microsoft can be trusted not to spy on their users and then share all the data with the NSA through the PRISM program - Oh wait...

**Reply by Asiachanel** [reply]
Ok nice

**Reply by markjpT** [reply]
> FastHub-Libre is another good open source client found on F-Droid.
> 
> 
> 
> Very disappointed to see GitHub hiding their source code. One can only wonder what they're trying to hide. 
> 
> Good thing Microsoft can be trusted not to spy on their users and then share all the data with the NSA through the PRISM program - Oh wait...

**Reply by XMA2l** [reply]
Fasthub-RE is on izzys repo now - to

**Comment by Asiachanel** [comment]
Gm day

**Comment by oniGino** [comment]
bump this

**Comment by yqs112358** [comment]
Interested in this to🧐 
As Github Mobile's UI design is a favorite of mine, it's sad that we cannot help contributing to it.

**Comment by Q1CHENL** [comment]
Microsoft what are you hiding bruh💀

**Comment by jcs090218** [comment]
I hope it will be open-source one day!

**Reply by Asiachanel** [reply]
53

**Comment by MdAshrafUllah** [comment]
No, the official GitHub mobile app is not open source. While GitHub itself is built upon open-source software like Git, and it provides open-source tools and libraries for developers, the mobile applications for iOS and Android are proprietary and not available for public inspection or modification.
